exporters/zipkin/src/main/java/io/opentelemetry/exporter/zipkin/ZipkinSpanExporterBuilder.java

```diff
@@ -159,8 +211,35 @@ public ZipkinSpanExporterBuilder setMeterProvider(MeterProvider meterProvider) {
   public ZipkinSpanExporter build() {
     Sender sender = this.sender;
     if (sender == null) {
+      if (trustedCertificatesPem != null) {
+        try {
+          X509TrustManager trustManager = TlsUtil.trustManager(trustedCertificatesPem);
+          X509KeyManager keyManager = null;
+          if (privateKeyPem != null && certificatePem != null) {
+            keyManager = TlsUtil.keyManager(privateKeyPem, certificatePem);
+          }
+          this.okHttpSenderBuilder
+              .clientBuilder()
+              .sslSocketFactory(TlsUtil.sslSocketFactory(keyManager, trustManager), trustManager);
+        } catch (SSLException e) {
+          throw new IllegalStateException(
+              "Could not set trusted certificate for Zipkin HTTP connection, are they valid X.509 in PEM format?",
+              e);
+        }
+      }
+
+      if (retryPolicy != null) {
+        this.okHttpSenderBuilder
+            .clientBuilder()
+            .addInterceptor(
+                new RetryInterceptor(
+                    retryPolicy,
+                    (response) ->
+                        RetryUtil.retryableHttpResponseCodes().contains(response.code())));
```

Comment:
Is retry valid for zipkin? Is there some sort of specification of which status codes are retryable for zipkin? The status codes in `RetryUtil.retryableHttpStatusResposneCodes()` are defined in the [OTLP specification](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/otlp.md#failures-1).

Reply:
Why would it be different between the two ? Both are HTTP status code

Reply:
> Why would it be different between the two ? Both are HTTP status code

Maybe zipkin uses those status codes differently and returns one in situations where a retry would not solve the problem.

Reply:
I removed the retry part, so I keep this PR focused only on TLS client.